src/GraphQL/AddElementToAreaMutation.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace DNADesign\Elemental\GraphQL;
+
+use InvalidArgumentException;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Definition\ResolveInfo;
+use SilverStripe\GraphQL\MutationCreator;
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\GraphQL\OperationResolver;
+use DNADesign\Elemental\Models\ElementalArea;
+use DNADesign\Elemental\Services\ReorderElements;
+
+class AddElementToAreaMutation extends MutationCreator implements OperationResolver
+{
+    public function attributes()
+    {
+        return [
+            'name' => 'addElementToArea',
+            'description' => 'Adds an Element to an ElementalArea, optionally after another Element'
+        ];
+    }
+
+    public function type()
+    {
+        return $this->manager->getType('Block');
+    }
+
+    public function args()
+    {
+        return [
+            'ClassName' => ['type' => Type::nonNull(Type::id())],
+            'ElementalAreaID' => ['type' => Type::nonNull(Type::id())],
+            'AfterElementID' => ['type' => Type::id()],
+        ];
+    }
+
+    public function resolve($object, array $args, $context, ResolveInfo $info)
+    {
+        $elementClass = $args['ClassName'];
+        $elementalAreaID = $args['ElementalAreaID'];
+        $afterElementID = isset($args['AfterElementID']) ? $args['AfterElementID'] : null;
+
+        if (!is_subclass_of($elementClass, BaseElement::class)) {
+            throw new InvalidArgumentException("$elementClass is not a subclass of " . BaseElement::class);
+        }
+
+        $elementalArea = ElementalArea::get()->byID($elementalAreaID);
+
+        if (!$elementalArea) {
```

Comment:
You should also check `$elementalArea->canEdit()` here, or some other kind of permission check using `$context['currentUser']`